fvtest/compilertest/CMakeLists.txt

```diff
@@ -54,11 +54,18 @@ elseif(OMR_ARCH_POWER)
 		p/codegen/Evaluator.cpp
 		${OMR_ROOT}/compiler/p/env/OMRDebugEnv.cpp
 	)
+elseif(OMR_ARCH_S390)
+	list(APPEND COMPILERTEST_FILES 
+                z/codegen/TestCodeGenerator.cpp
+		z/codegen/Evaluator.cpp
+		${OMR_ROOT}/compiler/z/env/OMRDebugEnv.cpp
+	)
 endif()
 
 create_omr_compiler_library(NAME       testcompiler 
                             OBJECTS    ${COMPILERTEST_FILES}
-							DEFINES JITTEST TEST_PROJECT_SPECIFIC PROD_WITH_ASSUMES)
+							DEFINES JITTEST TEST_PROJECT_SPECIFIC PROD_WITH_ASSUMES
+			                                INCLUDES ${OMR_ROOT}/fvtest)
```

Comment:
Is this the right change or should the `#include` should be changed similar to how it is in jitbuilder?

Reply:
Which `#include` in JitBuilder? IMO, includes in the compiler technology shouldn't include the project name. Now, that's not universally held, but my pref would be to alter the include path rather than alter the `#include` -- @0xdaryl may have a differing opinion here. 

In general, I'd like to get rid of the `#include "project/..."` includes.

Reply:
In `omr/fvtest/compilertest/z/codegen/TestCodeGenerator.hpp" there is the following include `#include "compilertest/codegen/TestCodeGenerator.hpp"`.  This include in my opinion should either be `#include "codegen/TestCodeGenerator.hpp"` if the include path is modified to have `omr/fvtest/compilertest' near the beginning or `#include "fvtest/compilertest/codegen/TestCodeGenerator.hpp"` since that is effectively how the include works in the similar file in jitbuilder.

Reply:
So the last one, I think is wrong. The first one though, I think is how it ought to be. 

```
#include "codegen/TestCodeGenerator.hpp" // < -- Right
#include "compilertest/codegen/TestCodeGenerator.hpp" // < -- wrong
#include "fvtest/compilertest/codegen/TestCodeGenerator.hpp" // < -- more wrong
```

@rbanerjee In light of the above, do you want to just change this PR to instead correct the include to the first line above?